pom.xml

```diff
@@ -3,18 +3,18 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-      <groupId>org.sonatype.oss</groupId>
-      <artifactId>oss-parent</artifactId>
-      <version>7</version>
+    <groupId>org.sonatype.oss</groupId>
+    <artifactId>oss-parent</artifactId>
+    <version>7</version>
   </parent>
 
   <groupId>com.urbanairship</groupId>
   <artifactId>connect-client</artifactId>
   <version>5.3.1-SNAPSHOT</version>
   <packaging>jar</packaging>
 
-  <name>Connect Client</name>
-  <description>The UA Connect Java client library</description>
+  <name>Real Time Data Stream Client</name>
```

Comment:
I wonder if it's going to cause trouble with it in Maven for us to rename the library itself... we should probably do some research on this before actually updating/deploying

Reply:
Also I think we would want to say "Real-Time Data Streaming"

Reply:
I guess it will only change the values in the name and description here: https://mvnrepository.com/artifact/com.urbanairship/connect-client/1.0.1
I guess if as long as we do not change the package name, it should not impact anything